engine/storage/datamotion/src/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -711,6 +711,7 @@ private void handleCreateVolumeFromSnapshotBothOnStorageSystem(SnapshotInfo snap
         }
         catch (Exception ex) {
             errMsg = ex.getMessage() != null ? ex.getMessage() : "Copy operation failed in 'StorageSystemDataMotionStrategy.handleCreateVolumeFromSnapshotBothOnStorageSystem'";
+            copyCmdAnswer = new CopyCmdAnswer(errMsg);
```

Comment:
@syed What I don't have in my version of this fix is this line of code. I can add it to mine.